hugo/assets/scss/components/site.scss

```diff
@@ -23,6 +23,7 @@
     &__content {
         display: flex;
         flex: 1 0 auto;
+        flex-direction: column;
```

Comment:
This was because of the separator in the single blog template, being outside of the article BEM. On a bigger screen this would result in the page flexing as row, instead of column.

But, since I put the separator in the article__footer, this problem is fixed. So this I will remove this styling.